### DISCUSSION TITLE
Deactivate analysis details

### DISCUSSION THREAD

**Discussion by mediakanal** [YetiForceCompany/YetiForceCRM#14254] [General] [answered]
Hello, how do I deactivate the analysis details in the My Mailbox module?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
from version 6.2 it will be possible to disable verification in the mail list 
https://github.com/YetiForceCompany/YetiForceCRM/blob/8c7effbe07e520da495aef5f0a5eef14e9505e58/config/Components/ConfigTemplates.php#L133-L136

**Reply by mediakanal** [reply]
Hello, I mean this message in the Roundcube client from CRM:
"

Vorsicht bei dieser Nachricht, die E-Mail-Adresse im Absender-Feld wurde gefälscht.
Klicken Sie daher keine Links, laden Sie keine Anhänge herunter und geben Sie keine persönliche Informationen als Antwort an.
Absender: Fehlerhaft
From: 
Return-Path: 
"
Can I switch off that this analysis is no longer performed?

Mit freundlichen Grüßen
Simon

**Comment by mediakanal** [comment]
Unfortunately, I still haven't managed to deactivate this message.
![Bildschirmfoto 2021-04-02 um 17 52 17](https://user-images.githubusercontent.com/79196539/113431901-bc873d80-93dc-11eb-9be5-0e563815ebd1.png)

**Reply by rioshaz** [reply]
follow this man :
https://github.com/YetiForceCompany/YetiForceCRM/discussions/13965#discussioncomment-253551

should help

**Reply by mariuszkrzaczkowski** [reply]
This has been rebuilt in the new version and it looks better
![image](https://user-images.githubusercontent.com/10198654/113449595-76db6c80-93fe-11eb-810d-718b28633988.png)
only after clicking it shows the details
![image](https://user-images.githubusercontent.com/10198654/113449617-83f85b80-93fe-11eb-855d-1c9c13f15db4.png)
